src/components/v2/values/common/ChartValuesSelectors.tsx

```diff
@@ -184,10 +184,7 @@ export const ChartRepoSelector = ({
             <components.MenuList {...props}>
                 {props.children}
                 <div
-                    className="flex react-select__bottom bcn-0 pt-10 pb-2"
-                    style={{
-                        bottom: '-4px',
-                    }}
+                    className="flex react-select__bottom bcn-0"
```

Comment:
is this change intentional? (not related to this story though)

Reply:
Yes. There was one visual/UI issue with the custom sticky bottom in react-select. There's some gap between the sticky bottom and the actual dropdown bottom causing text to be visible while scrolling through options. It's discovered as part of this story so fixed it.

The fix includes the removal of extra padding.